.github/actions/javascript/getGraphiteString/getGraphiteString.ts

```diff
@@ -1,6 +1,6 @@
-const core = require('@actions/core');
-const github = require('@actions/github');
-const fs = require('fs');
+import core from '@actions/core';
+import github from '@actions/github';
```

Comment:
I guess it should be [so](https://www.npmjs.com/package/@actions/core#import-the-package)
```suggestion
import * as core from '@actions/core'
import * as github from '@actions/github'
```